.github/pull_request_template.md

```diff
@@ -25,18 +29,18 @@ Automatic preview functionality is currently only available for some branches. F
 Additional information:
 <!--- Optional: Include additional context or expand the description here.--->
 
-
-
 <!--- Next steps after opening your PR:
 
-* Ask for peer review from the OpenShift docs team:
-  - For Red Hat associates: Ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
-    * A link to the PR.
-    * The size of the PR that the GitHub bot assigns (ex: XS, S, M, L, XL).
-    * If there is urgency or a deadline for the review.
+* Ask for review from the OpenShift docs team:
   - For community authors: Request a review by tagging @openshift/team-documentation in a GitHub comment.
-
-  Slack is the quickest and preferred way to request a review.
+  - For Red Hat associates:
+    * For normal peer requests, add a comment that contains this text: /label peer-review-needed
+    * For normal merge review requests, add a comment that contains this text: /label merge-review-needed
+    * For urgent requests, ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
```

Comment:
My only question here is that it could be an urgent peer review or an urgent merge review. I hate making it more complicated than needed, but wondering if it should also mention the @merge-review-squad instead for merging. wdyt?

Reply:
It's better to make it as complete as possible! I've added another update.